metapi/metapi.js

```diff
@@ -22,6 +24,9 @@ window.metapi = {};
     console.error("Failed to fetch API field mappings from MS API:", jqXhr);
   });
 
+  $.getScript("https://raw.githubusercontent.com/joewalnes/reconnecting-websocket/f8055b77ba75e5d564ffb50d20a483bdd7edccdf/reconnecting-websocket.min.js",
```

Comment:
I don't really like using rawgit since they're extremely aggressive with caching. I'm not sure what a more suitable alternative would be.